### ISSUE TITLE
EXCEPTION: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'sing100'

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1766]
on initial load when typing the URL (if i've already been there; my guess is that google cheats on the page load and something isn't registered / document isn't actually ready to go). Is there a way to like restart the app on initial load to reduce this possibly happening?

**Comment by btopro** [comment]
Ooooo @heyMP part of this is if the homepage is set to the app itself then it doesn't figure out base path correctly. It needs to read off current_path as a variable if that's possible instead of the URL itself.

**Comment by heyMP** [comment]
Oh yes, that makes sense.  The app is loading with no router information.  Yeah I think I might be able to implement something like that.  Or at least, forcing a url redirect if the router comes back with no information.

**Comment by btopro** [comment]
related to homepage, fixed
